### PR TITLE
[IA-40] incorrect arrangement of elements

### PR DESCRIPTION
**Pull request by otecMuxah**
Fix for enrollment plot.

this PR also includes minor fixes for : 
- [IA-39](https://youtrack.raccoongang.com/issue/IA-39) one line of code changed to fix it
- [IA-27](https://youtrack.raccoongang.com/issue/IA-27) minor fixes to gradebook

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -102,3 +102,5 @@ ENV/
 
 notes
 .idea/
+
+fake_views/
```
Comment (flying-pi) [inline .gitignore]: Maybe better to add this to ignorelist inside IDE instead global ignore?

Comment (otecMuxah) [reply .gitignore]: done

File: rg_instructor_analytics/static/rg_instructor_analytics/js/GradebookTab.js [hunk 2]
```diff
@@ -76,22 +76,29 @@ function GradebookTab(button, content) {
 
 
         $(greadebookTab.gradebookTableBody).click(function (element) {
+            let colorArray = greadebookTab.examNames.map((item, idx, arr)=>{
```
Comment (idegtiarov) [inline rg_instructor_analytics/static/rg_instructor_analytics/js/GradebookTab.js]: I would add spaces around `=>`

File: rg_instructor_analytics/static/rg_instructor_analytics/js/GradebookTab.js [hunk 3]
```diff
@@ -76,22 +76,29 @@ function GradebookTab(button, content) {
...
             var stat = {
                 y: greadebookTab.studentInfo[element.target.dataset['position']].grades,
                 x: greadebookTab.examNames,
                 type: 'bar',
                 marker:{
-                    color: ['#568ecc', '#568ecc','#568ecc','#568ecc','#568ecc','#c14f84']
+                    color: colorArray
                 },
-                width: 0.6,
+                width: 0.6
```
Comment (idegtiarov) [inline rg_instructor_analytics/static/rg_instructor_analytics/js/GradebookTab.js]: Why do you remove the comma, it as usual practice to have a comma at the end of some container content ends?

File: rg_instructor_analytics/static/rg_instructor_analytics/js/ProblemTab.js [hunk 4]
```diff
@@ -47,21 +47,23 @@ function ProblemTab(button, content) {
...
             }
             bars = `<ul class="plot-body">${bars}</ul>`;
-            //build y axis
+            //build x axis
             let axis = ``;
+            let small;
+            if (yAxis.length > 10) {small = 'small'}
             yAxis.forEach((item) => {
-                axis += `<li style="width: ${(100 - yAxis.length) / yAxis.length}%;"><div>${item}</div></li>`
+                axis += `<li class="hw-xaxis ${small}" style="min-width: ${(100 - yAxis.length) / yAxis.length}%;"><div>${item}</div></li>`
```
Comment (idegtiarov) [inline rg_instructor_analytics/static/rg_instructor_analytics/js/ProblemTab.js]: Could you please split this line into two sp review will be much easier.

**Review by idegtiarov — CHANGES_REQUESTED** [review]

File: rg_instructor_analytics/static/rg_instructor_analytics/js/GradebookTab.js (hunk 2 above)
Comment (otecMuxah) [reply rg_instructor_analytics/static/rg_instructor_analytics/js/GradebookTab.js]: done

File: rg_instructor_analytics/static/rg_instructor_analytics/js/ProblemTab.js (hunk 4 above)
Comment (otecMuxah) [reply rg_instructor_analytics/static/rg_instructor_analytics/js/ProblemTab.js]: done

**Review by idegtiarov — APPROVED** [review]

**Pull request merged**